### DISCUSSION TITLE
How to use system libs in manifest mode for transitive dependencies?

### DISCUSSION THREAD

**Discussion by msvetkin** [microsoft/vcpkg#25234] [Q&A] [answered]
I have vcpkg.json which is: 

```json
{
  "name": "test",
  "version-string": "0.0.1",
  "dependencies": [
    "paho-mqtt",
    "libdatachannel"
  ],
}
```

Both of this libraries ([libdatachannel](https://github.com/microsoft/vcpkg/blob/e78aa8421173e02315fabd88c7cd77de7a3c8e9a/ports/libdatachannel/vcpkg.json#L9), [paho-mqtt](https://github.com/microsoft/vcpkg/blob/e78aa8421173e02315fabd88c7cd77de7a3c8e9a/ports/paho-mqtt/vcpkg.json#L7)) depends on `openssl`.

Is there a way to tell vcpkg don't build `openssl` and use system provided?

`VCPKG_PREFER_SYSTEM_LIBS` affects only cmake `find_*` commands and stills tries to build `openssl`.

Currently I see only one way to solve it by making `openssl` as a feature for `libdatachannel` and `paho-mqtt` and then inside `vcpkg.json` disable default features for both of them.

Is there anything better what I can do right now? 
If there is not then it could be a nice feature to implement.

**Comment by Osyotr** [comment, marked as answer]
Create empty overlay port for openssl.
